lib/dictator.ex

```diff
@@ -10,6 +10,25 @@ defmodule Dictator do
   end
 
   defp authorize(conn, opts) do
+    action = conn.private[:phoenix_action]
+
+    cond do
+      opts[:only] ->
+        if action in opts[:only],
+          do: do_authorize(conn, opts),
+          else: conn
+
+      opts[:except] ->
+        if action in opts[:except],
+          do: conn,
+          else: do_authorize(conn, opts)
+
+      true ->
+        do_authorize(conn, opts)
+    end
+  end
```

Comment:
this feels like a `should_authorize?` function, tbh

Reply:
Donnest